cmd/crc/cmd/ip.go

```diff
@@ -15,8 +15,8 @@ func init() {
 
 var ipCmd = &cobra.Command{
 	Use:   "ip",
-	Short: "Get IP of the instance",
-	Long:  "Get IP of the instance",
+	Short: "Get IP address of the running cluster",
```

Comment:
I likely the notion of `running`, and the use of cluster seems more appropriate in general. @code-ready/crc-devel 

so, likely not instance, or VM, but cluster. (we have been discussing ths internally).